magpie/typedefs.py

```diff
@@ -86,3 +87,12 @@
     ConfigItem = Dict[Str, JSON]
     ConfigList = List[ConfigItem]
     ConfigDict = Dict[Str, Union[Str, ConfigItem, ConfigList, JSON]]
+
+    WebhookTemplateParameters = Dict[Str, AnyValue]
+    WebhookConfigSettings = TypedDict("WebhookConfigSettings", {
+        "name": Str, "action": Str, "method": Str, "url": Str, "format": Str, "payload": JSON
+    })
+    WebhookLoadedSettings = TypedDict("WebhookLoadedSettings", {
+        "name": Str, "method": Str, "url": Str, "format": Str, "payload": JSON
+    })
```

Comment:
Unsure here about the difference between WebhookConfigSettings and WebhookLoadedSettings, and the purpose of having 2 definition of settings?

Reply:
see other reply